// skip: Bug in unwinding past uncaught errors.
do
    do
        throw "unwind error"
    catch is Num then
        print("Should not be caught here.")
    end
catch is String then
    print("caught") // expect: caught
end
